src/components/search/detailed/AppSearchDrawer.js

```diff
@@ -137,10 +137,7 @@ function SearchToolbar(props) {
                 value={searchText}
                 onChange={handleInputChange}
                 className={classes.searchField}
-                InputProps={{
-                    startAdornment: <SearchIcon color="primary" />,
-                    disableUnderline: true,
-                }}
+                InputProps={{ startAdornment: <SearchIcon color="primary" /> }}
```

Comment:
This was throwing a warning that the DOM doesn't recognize `disableUnderline`, and removing it seemed to have no effect.